### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by nanavinay**
Changes in renderPeriods adding periods.name in Text component

### CODE REVIEW

**Comment by nanavinay** [comment]
in multi-period  should have prop called name should be nice to render an item in the calendar

File: src/calendar/day/multi-period/index.js [hunk 1]
```diff
@@ -70,7 +70,13 @@ class Day extends Component {
             marginRight: 4
           });
         }
-        return <View key={index} style={style}/>;
+        return (
+           <View key={index} style={style}>
+              validPeriods.map((period, index) => {
```
Comment (Inbal-Tish) [inline src/calendar/day/multi-period/index.js:75]: Why a map inside a map (line #52)?

File: src/calendar/day/multi-period/index.js [hunk 2]
```diff
@@ -70,7 +70,13 @@ class Day extends Component {
...
+              <Text style={{fontWeight:'600', fontSize:12}}>{period.name}</Text>
```
Comment (Inbal-Tish) [inline src/calendar/day/multi-period/index.js:76]: If you add a 'name' you should declare it, otherwise, no one will know that there is such an option.

**Comment by Inbal-Tish** [comment]
@nanavinay Please provide screenshots and a detailed description of your feature.

File: src/calendar/day/multi-period/index.js (hunk 2 above)
Comment (Inbal-Tish) [reply src/calendar/day/multi-period/index.js:76]: Also, inline style is bad for performance and you should provide a way to override it.

**Pull request closed without merging**